docs/standard/linq/sort-elements-multiple-keys.md

```diff
@@ -1,17 +1,20 @@
 ---
-title: "How to sort elements on multiple keys (C#)"
+title: How to sort elements on multiple keys - LINQ to XML
+description: Sort XML elements on multiple keys.
 ms.date: 07/20/2015
+dev_langs:
+  - "csharp"
+  - "vb"
 ms.assetid: 3b2760b6-d607-4ac7-b784-5c6524e2a0e0
 ---
-# How to sort elements on multiple keys (C#)
 
-This topic shows how to sort on multiple keys.
+# How to sort elements on multiple keys (LINQ to XML)
 
-## Example
+This article shows how to sort on multiple keys.
```

Comment:
In some cases, we could add the keywords C# or Visual Basic to the intro to make sure we don't lose the SEO that used to be in the title.

```suggestion
This article shows how to sort on multiple keys in C# or Visual Basic.
```